### ISSUE TITLE
Wrong indentation for a fortran snippet

### ISSUE THREAD

**Issue by kindlychung** [garbas/vim-snipmate#239]
I am having problems with this fortran snippet:

```
snippet impl
    implicit none
    $0
```

After `impl` and then tab:

```
program divide
  implicit none
^cursor here
end program divide
```

```
program divide
  implicit none
  ^cursor expected to be here
end program divide
```

**Comment by ajzafar** [comment]
@kindlychung This is a bug in the way SnipMate initially expands the snippet text into the buffer. Until I get this fixed, you can work around this by simply specifying some default placeholder text like so

    snippet impl
        implicit none
        ${0:text}

**Comment by ajzafar** [comment]
Alright, this is fixed in 2d70860. Thanks so much for the report!
